test/periphery/fork/merkle-lockup/MerkleLL.t.sol

```diff
@@ -93,7 +93,14 @@ abstract contract MerkleLL_Fork_Test is Fork_Test {
 
         // Sort the leaves in ascending order to match the production environment.
         MerkleBuilder.sortLeaves(leaves);
-        vars.merkleRoot = getRoot(leaves.toBytes32());
+
+        // Compute the Merkle root.
+        if (leaves.toBytes32().length == 1) {
```

Comment:
I had to add these conditions in the test because Murky requires the tree to have minimum 2 leaves.

Reply:
let's remove them

it is a mistake introduced here: 

https://github.com/sablier-labs/v2-core/blob/a97e5ae278e82a91533d42ae4df2cd3f95c14f81/test/periphery/fork/merkle-lockup/MerkleLL.t.sol#L61

it should be: `vm.assume(params.leafData.length > 1);`

commit when was introduced: https://github.com/sablier-labs/v2-core/commit/a97e5ae278e82a91533d42ae4df2cd3f95c14f81

Reply:
This is why I introduced that change https://github.com/sablier-labs/v2-periphery/issues/266

Reply:
completely missed it, sorry

let's leave it then like this